packages/vue/src/components/atoms/SfArrow/SfArrow.vue

```diff
@@ -1,22 +1,36 @@
-<template>
-  <SfButton class="sf-arrow" type="button" v-bind="$attrs" v-on="$listeners">
+<template functional>
+  <component 
```

Comment:
why  `component` and not simply `SfButton`?

Reply:
Because it's functional component and I cannot register custom component normally and I didn't want to use render function. That is why I used this workaround. I found it in this article https://dev.to/vhoyer/functional-components-in-vue-js-20fl

Reply:
@mayashavin  Wdyt? Maybe I overlooked something and it can be done in easier way?